docs/docs/userguide/configuration/JIRA_CONFIGURATION.md

```diff
@@ -18,6 +18,24 @@ For more information, see [Jira Version Tags](https://hub.docker.com/r/atlassian
 jira_version_tag = "<JIRA_VERSION_TAG>"
 ```
 
+### Number of Jira application nodes
+
+The initial Jira installation require to be started only with a single application node. After all the setup steps
```

Comment:
`The initial Jira installation need's to be started with a single application node.`